src/platform/w3dengine/client/w3ddisplay.cpp

```diff
@@ -881,21 +1374,136 @@ void W3DDisplay::Preload_Texture_Assets(Utf8String texture)
     }
 }
 
+#define BFT_BITMAP 0x4d42
```

Comment:
This corresponds to BM aka 'BM' (little endian) aka TwoCC<'M','B'>() (big endian)
I think we do not have a TwoCC function yet. Only rts::FourCC. But a TwoCC could be added in rtsutils.h
But I guess 0x4d42 is OK too. Could add a comment behind here `// BM`

https://docs.microsoft.com/en-us/windows/win32/api/wingdi/ns-wingdi-bitmapfileheader

Reply:
fixed.